packages/wiz/data_stream/vulnerability/elasticsearch/ingest_pipeline/default.yml

```
@@ -606,6 +606,17 @@ processors:
       field: observer.vendor
       tag: set_observer_vendor
       value: 'Wiz'
+  - set:
+      field: container.image.name
+      tag: set_continaer_name
+      copy_from: wiz.vulnerability.vulnerable_asset.name
+      if: ctx?.wiz.vulnerability?.vulnerable_asset?.type == 'CONTAINER_IMAGE'
```

```suggestion
      if: ctx.wiz.vulnerability?.vulnerable_asset?.type == 'CONTAINER_IMAGE'
```
`ctx` is always non-null.

---

I missed that `wiz` is not guarded. It will likely never be null, but it should be guarded anyway.